pycti/entities/opencti_case_incident.py

```diff
@@ -421,13 +415,13 @@ def contains_stix_object_or_stix_relationship(self, **kwargs):
                 "info",
                 "Checking StixObjectOrStixRelationship {"
                 + stix_object_or_stix_relationship_id
-                + "} in Case {"
+                + "} in CaseIncident {"
                 + id
                 + "}",
             )
             query = """
-                query CaseContainsStixObjectOrStixRelationship($id: String!, $stixObjectOrStixRelationshipId: String!) {
-                    caseContainsStixObjectOrStixRelationship(id: $id, stixObjectOrStixRelationshipId: $stixObjectOrStixRelationshipId)
```

Comment:
How does this query work by the way ? I don't see any query caseContainsStixObjectOrStixRelationship in opencti

Reply:
Apparently the query is missing in opencti so we will add it :) thank you for your feedback